onboarding/service_test.go

```diff
@@ -1,31 +1,30 @@
-package tenant_test
+package onboarding_test
 
 import (
 	"context"
 	"testing"
 	"time"
 
-	"github.com/influxdata/influxdb/v2/pkg/testing/assert"
-
 	"github.com/google/go-cmp/cmp"
-	"github.com/stretchr/testify/require"
-
 	"github.com/influxdata/influxdb/v2"
 	"github.com/influxdata/influxdb/v2/authorization"
 	icontext "github.com/influxdata/influxdb/v2/context"
 	"github.com/influxdata/influxdb/v2/kv"
+	"github.com/influxdata/influxdb/v2/onboarding"
+	"github.com/influxdata/influxdb/v2/pkg/testing/assert"
 	"github.com/influxdata/influxdb/v2/tenant"
 	influxdbtesting "github.com/influxdata/influxdb/v2/testing"
+	"github.com/stretchr/testify/require"
 )
 
 func TestBoltOnboardingService(t *testing.T) {
 	influxdbtesting.OnboardInitialUser(initBoltOnboardingService, t)
 }
 
 func initBoltOnboardingService(f influxdbtesting.OnboardingFields, t *testing.T) (influxdb.OnboardingService, func()) {
-	s := influxdbtesting.NewTestInmemStore(t)
+	s, closeBolt := influxdbtesting.NewTestBoltStore(t)
 	svc := initOnboardingService(s, f, t)
-	return svc, func() {}
+	return svc, closeBolt
```

Comment:
Do you happen to know what this ends up doing? Mostly curious as to how returning a no-op function previously worked at all...

Reply:
The no-op worked before because we were constructing & returning an inmem store, which doesn't have a `Close()` method. Now that we're using a bolt store we can return its closer instead.